android/src/main/java/com/tailscale/ipn/App.java

```diff
@@ -286,6 +285,17 @@ byte[] getPackageCertificate() throws Exception {
 		return null;
 	}
 
+	void requestNotificationPermission(Activity act) {
```

Comment:
Everything in /android has been forked /android_legacy.  It will need to be patched there to pick this up in our next client release.

Reply:
@barnstar just saw your comment in the linked issue. I'll update the PR unless you have the bandwidth to carry forward these changes? Either way, I am happy to help push this along.